server/athenian/api/openapi/openapi.yaml

```diff
@@ -1435,3 +1513,327 @@ components:
         merger:
           $ref: '#/components/schemas/DeveloperSet'
       type: object
+    FilterCommitsRequest:
+      description: Filter for listing commits.
+      example:
+        account: 1
+        date_from: 2000-01-23
+        date_to: 2020-05-23
+        in:
+        - '{1}'
+      allOf:
+      - $ref: '#/components/schemas/CommitFilter'
+      - properties:
+          property:
+            description: Main trait of the commits - the core of the filter.
+            enum:
+            - bypassing_prs
+            type: string
+        required:
+        - property
+        type: object
+    CommitsGroupedByTime:
+      description: Lists of commits for each time interval.
+      example:
+        include:
+          users:
+            github.com/gkwillie:
+              avatar: https://avatars0.githubusercontent.com/u/60340680?v=4
+        data:
+        - date: 2000-01-23
+          commits:
+          - repository: github.com/athenianco/athenian-api
+            hash: 79f0bccb856b35ef08ec93da2ff5585da4d2e50e
+            author:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+            committer:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
```

Comment:
Why not add this repeated fields into `included` fields as done with the user `avatar` at #107?

Reply:
Some commits do not have mapped usernames. I can do what you propose if you are fine with all the three props being optional.

Reply:
I already to be aware of a user not having an avatar, so the same about name or email.
Anyhow, what do you mean with "three props"? I'm only talking about `name` and `email`, since `login` would be the key to get the included user, and `timestamp` is unique per commit.

Reply:
There are commits outside of the GitHub userbase.

Reply:
Would that mean that in some cases there would be users with only `login` and under included there would be its `name` and `email`, and for other committers, there will be no `login` but only `name` and `email`, and nothing in the included?
If that's what you mean, lgtm

Reply:
Yes, exactly, you got it :+1:

Reply:
did you finally added `name` and `email` into de included fields?

Reply:
Now everything is optional, as we 🤝.

Reply:
Optional is one thing. Moving  `name` and `email` into de included fields another. This last one is a good way to save a lot of bandwidth as done with user avatars.

Reply:
Once we need it, I will surely list them, yes.